anitya/tests/test_flask.py

```diff
@@ -530,6 +565,75 @@ def test_projects_search(self):
             b'exact match, redirecting</li>'
         self.assertTrue(expected in output.data)
 
+    def test_logout(self):
+        """Assert the logout logouts user"""
+        with login_user(self.flask_app, self.user):
+            output = self.app.get('/logout')
+            self.assertEqual(output.status_code, 302)
+            self.assertEqual(
```

Comment:
This asserts the logout endpoint redirects the user to /, but nothing about the state of the user's login.

Reply:
Yeah, this should be here, I'm just need to look how to test this.